src/reference/informative/informA2_2.rst

```diff
@@ -0,0 +1,71 @@
+.. _inform2_2:
+
+.. container:: toggle
+
+  .. container:: header
+
+    See more
+
+  .. container:: infospec
+
+    .. container:: heading3
+
+      Specific information items
+
+    There are different kinds of information stored in XML elements.  In the example below, the :code:`<food_ideas>` element has 
+    In the example below:
+      - :code:`title`, :code:`instructions`, :code:`ingredient` have *text* content (note that numbers like 1.0 are treated as text),
+      - :code:`ingredients` and :code:`food_ideas` have *element* contents, because they have child elements inside them,
+      - :code:`category` is an *attribute* of the :code:`recipe` element, and :code:`name` and :code:`units` are *attributes* of the :code:`ingredient` element, and
+      - the note within :code:`<!-- ... -->` comment tags is an *unparsed* and is ignored.
```

Comment:
- Remove "an" in "an unparsed".
- Why is "unparsed" in italics and not "ignored"?

Reply:
See the correction.  The sentence should have read:
> the note within `<!-- ... -->` comment tags is of an *unparsed* type and is ignored.

The "unparsed" refers to the information types listed in the main text.  "Ignored" is nothing special, so needs no emphasis.

Reply:
473ee6c